Gemfile.lock

```diff
@@ -445,6 +459,7 @@ DEPENDENCIES
   guard-rspec
   jbuilder (~> 2.5)
   jquery-rails (~> 4.3, >= 4.3.1)
+  kaminari
```

Comment:
Are we currently using  `kaminari` in this PR?

Reply:
Kaminari is needed for filterrific :)